src/lib/utils.ts

```diff
@@ -0,0 +1,32 @@
+import jwt from 'jsonwebtoken'
+import { ManagementAppResponse } from './types'
+
+// Functions for interacting with the Management App
+const generateToken = (): string => {
+    const privateKey: string | undefined = process.env.MANAGEMENT_APP_PRIVATE_KEY
+    const memberId = 'openstax' // TODO: make depend on who's asking (post pilot)
```

Comment:
In the trusted app, we implemented as follow. 
`const memberId: string | undefined = process.env.MANAGEMENT_APP_MEMBER_ID`
And on line `10` below, we could have `if(privateKey && memberId)`

Reply:
We were inspired by the TOA! 🥇  But I held off on using the `MANAGEMENT_APP_MEMBER_ID` environment variable because 1) I wasn't sure where/if this value would be supplied in the environment, and 2) I didn't think the member ID "belonged" to the management app. I could be wrong :p Maybe it will come from the enclave environment? In which case `ENCLAVE_MEMBER_ID` makes more sense to me as a var name (to get pedantic).

Reply:
It makes completely sense. The variable can be passed to the `TOA` container by the Setup App which is not the case which is unlikely for the Setup App. Thanks for clarifying. 

PR Looks good to me. 💯

Reply:
:thinking: -- Maybe I'm confused, but wouldn't / couldn't we pass to both via the IaC? Something like the following [here](https://github.com/safeinsights/iac/blob/98909628524e0535db6a22b9af7ba4363682794e/secure-enclave/stack.ts#L272), for example:

```patch
             environment: {
                 ECS_CLUSTER: this.cluster.clusterArn,
                 BASE_TASK_DEFINITION_FAMILY: researchContainerTaskDef.family,
                 SECURITY_GROUP: this.researchContainerSg.securityGroupId,
                 VPC_SUBNETS: this.subnetsIsolated.map((subnet) => subnet.subnetId).join(','),
+                ENCLAVE_MEMBER_ID: 'openstax'
```

Reply:
It can be done that way but are we assuming that value is read from an env variable during deployment like this?
` ENCLAVE_MEMBER_ID:  process.env.ENCLAVE_MEMBER_ID`